.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Integration Test
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+jobs:
+  integration_test:
+    name: integration test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Set up python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Run Minimum Dependency Generator with 1 file
+        id: min_dep_gen_1
+        uses: ./
+        with:
+          requirements_paths: 'requirements.txt'
+      - name: Save the output minimum test requirements
+        run: printf "${{ steps.min_dep_gen_1.outputs.min_reqs }}" >> generated-min--test-reqs.txt
+      - name: Verify output
+        run: |
+          echo "${{ steps.min_dep_gen_1.outputs.min_reqs }}" | grep -c "packaging==20.9"
+          echo "${{ steps.min_dep_gen_1.outputs.min_reqs }}" | grep -c "requests==2.25.1"
+          echo "${{ steps.min_dep_gen_1.outputs.min_reqs }}" | grep -c "wheel==0.36.2"
+      - name: Run Minimum Dependency Generator with 2 files
+        id: min_dep_gen_2
+        uses: ./
+        with:
+          requirements_paths: 'test-requirements.txt requirements.txt'
+      - name: Save the output minimum test and core requirements
+        run: printf "${{ steps.min_dep_gen_2.outputs.min_reqs }}" >> generated-min-reqs.txt
+      - name: Verify output
+        run: |
+          echo "${{ steps.min_dep_gen_2.outputs.min_reqs }}" | grep -c "packaging==20.9"
+          echo "${{ steps.min_dep_gen_2.outputs.min_reqs }}" | grep -c "pytest==6.2.4"
+          echo "${{ steps.min_dep_gen_2.outputs.min_reqs }}" | grep -c "isort==5.8.0"
+          echo "${{ steps.min_dep_gen_2.outputs.min_reqs }}" | grep -c "pytest-cov==2.11.1"
```

Comment:
Would this step fail if the output does not match?

Reply:
Yep it would. 
I did a test commit:
- https://github.com/FeatureLabs/minimum-dependency-generator/pull/2/commits/bb3501a581914af3a138fc9b44dc0ac63a36c581

The CI failed as expected:
<img width="420" alt="Screen Shot 2021-05-06 at 8 17 53 PM" src="https://user-images.githubusercontent.com/8726321/117380648-34e39180-aea8-11eb-99e7-6e2a6609cf14.png">

Reply:
Cool, thanks!